apps/analysis_framework/schema.py

```diff
@@ -97,6 +98,7 @@ class AnalysisFrameworkDetailType(AnalysisFrameworkType):
     secondary_tagging = DjangoListField(WidgetType)  # Without section
     members = DjangoListField(AnalysisFrameworkMembershipType)
     preview_image = graphene.Field(FileField)
+    visible_projects = DjangoListField(ProjectType)
```

Comment:
We may need to create a strict project type for this.
cc: @AdityaKhatri

Reply:
Yeah, let's do it.
Lets only expose `title`, `id` & `isPrivate`.